app/helpers/api/integrations_helper.rb

```diff
@@ -24,18 +24,6 @@ def api_backend_hint(api_backend)
     t("formtastic.hints.proxy.api_backend_#{scheme}")
   end
 
-  def different_from_current?
-    true #TODO: implement method
```

Comment:
I'd be happy to implement, if I knew what it was meant for :sweat_smile: 
But it doesn't seem to be used anywhere.

Reply:
Maybe the idea was:

1. Detect when policies and metrics are not yet deployed to staging apicast. But we already solved this, and a "warning" icon appears on the "Integration -> Configuration" menu item when there are differences.
3. Compare staging with production apicast. But this is solved also, because the "Promote to production" button is only enabled when there are differences.

I think we can remove this.